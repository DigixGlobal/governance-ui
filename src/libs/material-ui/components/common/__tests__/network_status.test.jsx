import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NetworkStatus from '../network_status';

describe('<NetworkStatus />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <NetworkStatus
        name
        network={{
          name: 'test network',
        }}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
