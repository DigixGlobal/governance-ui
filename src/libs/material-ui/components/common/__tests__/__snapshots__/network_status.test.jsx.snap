// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NetworkStatus /> renders correctly 1`] = `
<Label
  content="test network"
  icon="remove"
  style={
    Object {
      "float": "right",
    }
  }
/>
`;
